examples/experimental-amp/pages/cat/index.amp.js

```diff
@@ -0,0 +1 @@
+export { default } from './index'
```

Comment:
Let's adjust these examples to show just `.js`, just `.amp.js` and then a combination of both for a single page?

Reply:
Updated